manifests/charts/istio-control/istio-discovery/Chart.yaml

```diff
@@ -6,6 +6,8 @@ tillerVersion: ">=2.7.2"
 description: Helm chart for istio control plane
 keywords:
   - istio
+  - istiod
```

Comment:
Can you comment on why these two lines are needed?

Reply:
They are added keywords to better the visibilty and searchability of the Chart